tasks/go.py

```diff
@@ -396,7 +397,20 @@ def check_mod_tidy(ctx, test_folder="testmodule"):
 
 @task
 def tidy_all(ctx):
-    for mod in DEFAULT_MODULES.values():
+    print(color_message('This command is deprecated, please use `tidy` instead', "orange"))
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ It would be great to have this on stderr